lib/pbench/client/oidc_admin.py

```diff
@@ -0,0 +1,148 @@
+from http import HTTPStatus
+import json
+
+import requests
+
+from pbench.server.auth import Connection
+
+
+class OIDCAdmin(Connection):
+    OIDC_REALM = "pbench-server"
+    OIDC_CLIENT = "pbench-dashboard"
+    ADMIN_USERNAME = "admin"
+    ADMIN_PASSWORD = "123"
+
+    def __init__(self, server_url: str):
+        super().__init__(server_url, verify=False)
+
+    def get_admin_token(self) -> dict:
+        """pbench-server realm admin user login.
+
+        Returns:
+            access_token json payload
+
+            { 'access_token': <access_token>,
+              'expires_in': 60, 'refresh_expires_in': 1800,
+              'refresh_token': <refresh_token>,
+              'token_type': 'Bearer',
+              'not-before-policy': 0,
+              'session_state': '8f558797-50e7-496d-bb45-3b5ac9fdcddb',
+              'scope': 'profile email'}
+
+        """
+        url_path = "/realms/master/protocol/openid-connect/token"
+        data = {
+            "grant_type": "password",
+            "client_id": "admin-cli",
+            "username": "admin",
+            "password": "admin",
+        }
+        return self.post(path=url_path, data=data).json()
+
+    def create_new_user(
+        self,
+        username: str,
+        email: str,
+        password: str,
+        first_name: str = "",
+        last_name: str = "",
+    ) -> requests.Response:
+        """Creates a new user under the OIDC_REALM and assign OIDC_CLIENT_ROLE
```

Comment:
I don't see anything in this code about assigning a role. Is this comment out of date? (And, while I'm reviewing a docstring, I might as well pass on what Webb said to me recently, that apparently we're supposed to be following a docstring convention of a single title line followed by a blank and a description. Yeah, I didn't know that too.)

Reply:
Yes, I forgot to update this comment and it remained there from when I was testing a couple of things.